app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,172 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch all forms under the admin access.
+ *    security:
+ *      - bearerAuth: []
+ *      - basicAuth: []
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminsFormGetEx'
+ *      '403':
+ *        $ref: '#/components/responses/Forbidden'
+ *      '401':
+ *        $ref: '#/components/responses/NoFormAccess'
+ *      '404':
+ *        $ref: '#/components/responses/ResourceNotFound'
+ */
 routes.get('/forms', async (req, res, next) => {
   await controller.listForms(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/forms/{formId}:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    description: This endpoint will fetch the form details(and metadata for versions).
+ *    security:
+ *      - bearerAuth: []
+ *      - basicAuth: []
+ *      - openId: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: ID of the form.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
+ *    responses:
```

Comment:
How much time/effort do we want to put into the docs for this sprint? For example, this endpoint also has a 404 response - I think a lot of the old docs don't mention the 404 response.